server/game/player.js

```diff
@@ -310,7 +310,14 @@ class Player extends Spectator {
             card.moveTo('draw deck');
             this.drawDeck.push(card);
         });
+        this.cardsInPlay.each(card => {
```

Comment:
Probably should move things out of discard and dead pile as well, because someone somewhere is going to do that.